### DISCUSSION TITLE
Extended modules migrations

### DISCUSSION THREAD

**Discussion by Yasaie** [OCA/OpenUpgrade#2939] [Q&A] [answered]
Hi everyone,

I have a custom module in odoo 13.0 that now become 2 separate modules in 14.0 and I need to migrate related data across both modules.
There is a merged_modules in apriori to get analysis and data but not in reverse for extended_modules.
Do you have any ideas what should I do in this kind of situations?
Would you please reference me to an example module?

**Comment by pedrobaeza** [comment, marked as answer]
Just install the new module in the target database.

Example: This one didn't exist in v11. https://github.com/OCA/social/tree/12.0/mail_attach_existing_attachment_account

**Reply by Yasaie** [reply]
I had a module in 13.0 for example `module_foo` that now become `module_foo` and `module_bar` in 14.0. 
I need to move some tables, fields and xmls that transferred and some changed from `module_foo` to `module_bar`. 
How do I install `module_bar` while upgrading `module_foo` so I don't have any data loss?

**Reply by pedrobaeza** [reply]
As said, just install `module_bar` when the upgrade is completed. You won't lose anything.
